backend/game.py

```diff
@@ -281,8 +243,11 @@ def play(self):
             except InvalidMove as move:
                 print(self)
                 print('{} '.format(move))
-
-        print('Goodbye :)\n')
+        # at this point, self.is_game_over()
+        print("{} wins!".format(self.is_game_over()))
```

Comment:
would be better to save the return value of `is_game_over` and then check it, since this does the computation over again

Reply:
Since it's in the while loop, either this isn't a problem or we need to make a variable which is updated in the course of the loop